scripts/Hello_World_Klotzey13.ps1

```diff
@@ -0,0 +1 @@
+Write-Host "Hello World!!!"
```

Comment:
Please add comments to the file which contain the required info. You will find which info you need in issue #162.